src/components/headers/AppMenu.js

```diff
@@ -31,10 +53,18 @@ const menuItemStyles = css`
 
 const MenuWrapper = styled.div`
   align-items: center;
+  background-color: ${(props) => (props.isSticky ? NEUTRALS.WHITE : NEUTRALS.GRAY_05)};
+  box-shadow: ${(props) => (props.isSticky ? '0 5px 20px rgba(0, 0, 0, 0.05)' : 'none')};
   display: flex;
   justify-content: space-between;
-  margin: 0;
+  margin: ${(props) => (props.isSticky ? '0 -32px' : '0')};
   max-width: 100%;
+  padding: ${(props) => (props.isSticky ? '16px 32px' : '16px 0')};
+  position: ${(props) => (props.isSticky ? 'fixed' : 'relative')};
+  top: 0;
+  left: ${(props) => (props.isSticky ? '32px' : 'auto')};
+  right: ${(props) => (props.isSticky ? '32px' : 'auto')};
+  z-index: 100;
```

Comment:
I think I would have written the prop-dependent attributes like this:

```
${(props) => props.isSticky && css`
     background-color: ${NEUTRALS.WHITE};
     box-shadow: 0 5px 20px rgba(0, 0, 0, 0.05);
     margin: 0 -32px;
     padding: 16px 32px;
     position: fixed;
     left: 32px;
     right: 32px;
  `}

  ${(props) => !props.isSticky && css`
     background-color: ${NEUTRALS.GRAY_05};
     box-shadow: none;
     margin: 0;
     padding: 16px 0;
     position: relative;
     left: auto;
     right: auto;
  `}
```
Looks more readable to me. This is probably a personal preference though.

Reply:
I agree with alfonce about the readability concern. Don't make it two separate functions tho, a simple if...else or ternary would suffice.

Reply:
totally agree with y'all—this is painful to look at. i'll update.